manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ASUS.GPUTweakIII
+PackageVersion: 1.5.8.0
+Installers:
+- InstallerUrl: https://dlcdnets.asus.com/pub/ASUS/Graphic%20Card/Unique_Applications/GPU-Tweak-III-v1580.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: burn
+  NestedInstallerFiles:
+  - RelativeFilePath: GPU Tweak III Install.exe
+  InstallerSha256: 25A0D59345C039C581D7B670F23E9572937EA5D3E94610AF1CFF9B907A2D4642
+  InstallerSuccessCodes: 3010

```

Comment:
```suggestion
  InstallerSuccessCodes: 
  - 3010
```

Reply:
Apologies - I failed to test the previous suggestion. This has been tested to ensure it works.